README.md

```diff
@@ -11,9 +11,12 @@ automation and interaction capabilities for Infrastructure as Code (IaC) develop
 - **Module Search & Analysis**: Search and retrieve detailed information about Terraform modules
 - **Registry Integration**: Direct integration with Terraform Registry APIs
 - **Container Ready**: Docker support for easy deployment
+- **Security**: Origin validation to prevent cross-origin attacks
```

Comment:
Unnecessary with the security not below, in fact can you also remove the container ready point above please? it's not really a core feature

Reply:
sounds good!